sentry.properties

```diff
@@ -0,0 +1,3 @@
+defaults.url=https://sentry.io/
+defaults.org=london-borough-of-hackney
+defaults.project=bonuscalc-frontend
```

Comment:
There's also an auth token which can go in the git ignored `.sentryclirc` file which looks like this:

```
[auth]
token=***secret***
```

However it shouldn't be needed as the CI is responsible for uploading a release's source maps.